src/Web/UIEvent/InputEvent/InputTypes.purs

```diff
@@ -0,0 +1,155 @@
+module Web.UIEvent.InputEvent.InputTypes where
```

Comment:
We don't tend to pluralise in module names, so I'd just call this `Web.UIEvent.InputEvent.InputType`.